core/primitives-core/src/config.rs

```diff
@@ -536,6 +536,9 @@ pub enum ExtCosts {
     alt_bn128_g1_sum_base,
     #[cfg(feature = "protocol_feature_alt_bn128")]
     alt_bn128_g1_sum_byte,
+
+    // NB: this should be the last element of the enum.
```

Comment:
What is NB?

Reply:
[Note Bene](https://en.wikipedia.org/wiki/Nota_bene). I guess it's better to change this to Note.